shopelectro/selenium/pages/order.py

```diff
@@ -9,16 +13,37 @@ class OrderPage(Page):
 
     def __init__(self, driver):
         super().__init__(driver)
+        self.submit_button = Button(self.driver, (By.ID, 'submit-order'))
 
     @property
     def path(self):
         return CustomPage.objects.get(slug='order').url
 
-    def fill_contacts(self, contacts):
-        raise NotImplementedError
+    def fill_contacts(
+        self, name='Name', city='Санкт-Петербург', phone='2222222222', email='test@test.test',
+    ):
+        contacts = {
+            'id_name': name,
+            'id_city': city,
+            'id_phone': phone,
+            'id_email': email,
+        }
 
-    def make_order(self):
-        raise NotImplementedError
+        for id_, value in contacts.items():
+            Input(self.driver, (By.ID, id_)).send_keys(value)
 
-    def select_payment_type(self):
-        raise NotImplementedError
+    def make_order(self):
```

Comment:
It seems something went wrong and i can see the suggested change

Reply:
@artemiy312 , sorry, fixed my comment above

Reply:
`order` may have too many means: some representation of an order, order of positions, order as an action. In other words, it may looks both like a command and a query, so i'd like to leave the explicit name